apps/e2e/cypress/page_objects/basePage.js

```diff
@@ -48,7 +48,7 @@ export class Page {
 		summaryListActions: '.govuk-summary-list__actions',
 		summaryListKey: '.govuk-summary-list__key',
 		summaryListValue: '.govuk-summary-list__value',
-		summaryErrorMessages: '.govuk-error-summary [href="#msg"]',
+		summaryErrorMessages: '#main-content > div:nth-child(1) > div > div > div > div > ul > li > a',
```

Comment:
This seems rather verbose - does the error summary class selector not work?

Reply:
Understood, but I think we want to avoid such a specific and brittle selector.

I think `.govuk-list` on the other hand isn't specific enough, there could be another list on the page? Perhaps `.govuk-error-summary__list`? See https://design-system.service.gov.uk/components/error-summary/

Reply:
We should reference the error summary class rather than the general list class - that matches the error summary component then.

Reply:
As we discussed updated the selector(.govuk-list.govuk-error-summary__list)